packages/moonstone/VideoPlayer/MediaControls.js

```diff
@@ -664,9 +664,11 @@ const MediaControlsDecorator = hoc((config, Wrapped) => {
 
 				// Readout 'more' or 'back' button explicitly.
 				let selectedButton = Spotlight.getCurrent();
-				if (selectedButton) {
+				if (selectedButton === this.mediaControlsNode.querySelector(`.${css.moreButton}`)) {
```

Comment:
Decided to check to whether more button is focused to separate the readout case and moving focus to more button.

Note that the focus will move to "more" button when back key is pressed no matter where it was focused, including `leftComponents`, `rightComponents` while showing more controls.